catalog/core/catalog-core-commands/src/test/groovy/org/codice/ddf/commands/catalog/ExportCommandSpec.groovy

```diff
@@ -0,0 +1,283 @@
+package org.codice.ddf.commands.catalog
+
+import ddf.catalog.CatalogFramework
+import ddf.catalog.data.BinaryContent
+import ddf.catalog.data.Metacard
+import ddf.catalog.data.impl.AttributeImpl
+import ddf.catalog.data.impl.MetacardImpl
+import ddf.catalog.data.impl.ResultImpl
+import ddf.catalog.filter.proxy.builder.GeotoolsFilterBuilder
+import ddf.catalog.operation.QueryRequest
+import ddf.catalog.operation.ResourceRequest
+import ddf.catalog.operation.impl.QueryResponseImpl
+import ddf.catalog.operation.impl.ResourceResponseImpl
+import ddf.catalog.resource.ResourceNotFoundException
+import ddf.catalog.resource.impl.ResourceImpl
+import ddf.catalog.transform.MetacardTransformer
+import groovy.xml.MarkupBuilder
+import org.apache.karaf.shell.api.console.Session
+import org.osgi.framework.BundleContext
+import org.osgi.framework.ServiceReference
+
+import javax.activation.MimeType
+import java.nio.file.Paths
+import java.util.zip.ZipFile
+
+class ExportCommandSpec extends spock.lang.Specification {
+
+    ExportCommand exportCommand
+
+    CatalogFramework catalogFramework
+
+    MetacardTransformer xmlTransformer
+
+    File tmpHomeDir
+
+    void setup() {
+        tmpHomeDir = File.createTempDir()
+        System.setProperty("ddf.home", tmpHomeDir.canonicalPath)
+
+        ServiceReference xmlTransformerReference = Mock(ServiceReference)
+
+        xmlTransformer = Mock(MetacardTransformer)
+
+        xmlTransformer.transform(_ as Metacard, _ as Map) >> { metacard, map ->
+            metacardToXmlTransformer(metacard, map)
```

Comment:
❓ Instead of having a closure that does a "real" transformation, can't we just have the `transform` method return canned responses? That would eliminate the chance of having to deal with bugs in the `metacardToXmlTransformer` closure.